test/e2e/explorer.feature

```diff
@@ -52,3 +52,27 @@ Feature: Explorer page
     And I fill in "1465651642158264047" to "search bar input" field
     When I click "search bar button"
     Then I should see ID "1465651642158264047" in transaction header
+
+  Scenario: should show delegate statistics while being logged in
+    Given I'm logged in as "genesis"
+    And I wait 1 seconds
+    When I click "explorer" menu
+    When I fill in "2581762640681118072L" to "search input" field
+    And I click "input search button"
+    And I wait 3 seconds
+    And I click "delegate statistics"
+    And I wait 1 seconds
+    Then I should see 2 instances of "votersFilterQuery row"
+
+  Scenario: should show delegate statistics while being logged in
+    Given I'm logged in as "genesis"
+    And I wait 1 seconds
+    When I click "explorer" menu
+    When I fill in "4401082358022424760L" to "search input" field
+    And I click "input search button"
+    And I wait 3 seconds
+    And I click "delegate statistics"
+    And I wait 1 seconds
+    Then I should see 35 instances of "votesFilterQuery row"
+    When I click "show votes"
+    Then I should see 101 instances of "votesFilterQuery row"
```

Comment:
IMO there should be no new scenario. It should be part of the previous scenarios, to avoid most of the setup. You can even use `4401082358022424760L` instead of `16313739661670634666L` from the other scenarios.